addons/core/functions/fnc_getLrSettings.sqf

```diff
@@ -86,6 +86,18 @@ if (isNil "_radioCode") then {
     _value set [TFAR_CODE_OFFSET, _radioCode];
 };
 
+private _halfDuplexOverride = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
Yeah. You could also use stuff like scriptNull as default value and then do a isNull check. I did that somewhere else already.
But doesn't matter then.